specifyweb/specify/tree_views.py

```diff
@@ -51,6 +51,23 @@ def tree_view(request, treedef, tree, parentid, sortfield, includeAuthor=False):
     child_id = getattr(child, node._id)
     treedef_col = getattr(node, tree_table.name + "TreeDefID")
     orderby = tree_table.name.lower() + '.' + sortfield
+    
+    """
+        If the request wants to include an author in the response, we use a case to determine
+        if the rankId is >= the supplied rank, and if so then we add node.author to the query statement.
+        If the tree is not the taxon tree or the rankId is less than the supplied rank, add NULL instead.
+
+        For more information, view the sqlalchemy docs on case:
+            https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.case
+    """
+    includeAuthorAfterRank= int(request.GET.get('includeauthorafterrank')) if 'includeauthorafterrank' in request.GET else 99_999
```

Comment:
Can you rename this to `includeAuthor`? No need to be more verbose

Reply:
Fixed in https://github.com/specify/specify7/pull/2818/commits/b4a084f0cfdb2c133782f709cb8be62e5cef98a0

Reply:
Fixed in https://github.com/specify/specify7/pull/2818/commits/8bdc5b3e50c4355941ff7e544e53d2fb332f7e9f